components/dashboard/src/teams/variables/NamedOrganizationEnvvarItem.tsx

```diff
@@ -0,0 +1,163 @@
+/**
+ * Copyright (c) 2025 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { OrganizationEnvironmentVariable } from "@gitpod/public-api/lib/gitpod/v1/envvar_pb";
+import { useCallback, useState } from "react";
+import { OrganizationRemoveEnvvarModal } from "./OrganizationRemoveEnvvarModal";
+import { InputField } from "../../components/forms/InputField";
+import { ReactComponent as Stack } from "../../icons/Repository.svg";
+import { Button } from "@podkit/buttons/Button";
+import { useCreateOrganizationEnvironmentVariable } from "../../data/organizations/org-envvar-queries";
+import Modal, { ModalBody, ModalFooter, ModalFooterAlert, ModalHeader } from "../../components/Modal";
+import { TextInputField } from "../../components/forms/TextInputField";
+import { useToast } from "../../components/toasts/Toasts";
+import { LoadingButton } from "@podkit/buttons/LoadingButton";
+
+type Props = {
+    disabled?: boolean;
+    organizationId: string;
+    name: string;
+    variable: OrganizationEnvironmentVariable | undefined;
+};
+export const NamedOrganizationEnvvarItem = ({ disabled, organizationId, name, variable }: Props) => {
+    const [showRemoveModal, setShowRemoveModal] = useState<boolean>(false);
+    const [showAddModal, setShowAddModal] = useState<boolean>(false);
+
+    const value = variable ? "*****" : "not set";
+
+    return (
+        <>
+            {variable && showRemoveModal && (
+                <OrganizationRemoveEnvvarModal
+                    variable={variable}
+                    organizationId={organizationId}
+                    onClose={() => setShowRemoveModal(false)}
+                />
+            )}
+
+            {showAddModal && (
+                <AddOrgEnvironmentVariableModal
+                    organizationId={organizationId}
+                    staticName={name}
+                    onClose={() => setShowAddModal(false)}
+                />
+            )}
+
+            <InputField disabled={disabled} className="w-full max-w-lg">
+                <div className="flex flex-col bg-gray-50 dark:bg-gray-800 p-3 rounded-lg">
```

Comment:
Wanted to suggest to use `bg-pk-content-secondary` here, but the other cards in org settings use the colors you use here as well, so it would make it look weird and inconsistent, although the colors are pretty similar. Hence, I made CLC-1097, through which I'd like to change this everywhere under org settings as a bit more organized effort.